qual/src/main/java/com/virtuslab/qual/guieffect/NotUIThreadSafe.java

```diff
@@ -0,0 +1,4 @@
+package com.virtuslab.qual.guieffect;
+
+// TODO (#3252): [checker-framework][@Heavyweight]
```

Comment:
Whoops I didn't think of referencing other repos `TODOs`... maybe patch scripts/enforce-issue-number-for-todos so that sth like `// TODO (typetools/checker-framework#3252):@Heavyweight` is possible